src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -14,12 +14,12 @@ const testConfig = createTestConfig(
 
     dataSets: [
       'full-781-781a-8940-test.json',
-      'maximal-test',
+      // 'maximal-test',
       'maximal-bdd-test',
       'minimal-test',
       'minimal-bdd-test',
-      'newOnly-test',
-      'secondary-new-test.json',
+      // 'newOnly-test',
+      // 'secondary-new-test.json',
```

Comment:
What axe errors are showing up that require disabling these tests?

Reply:
You can view the complete log to get this:

http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/13647-cypress-axe-fix/5/pipeline/

```
[2020-09-21T20:32:06.027Z]   Running:  [90mapplications/disability-benefits/all-claims/tests/all-claims.cypress[39m           [90m(5 of 48)[39m
[2020-09-21T20:32:06.027Z]             [90m.spec.js[39m                                                                                
[2020-09-21T20:32:06.585Z] [info] [webpackbar] Compiling Webpack
[2020-09-21T20:32:11.816Z] [success] [webpackbar] Webpack: Compiled successfully in 4.83s
[2020-09-21T20:32:15.967Z] 
[2020-09-21T20:32:15.967Z] [0m[0m
[2020-09-21T20:32:15.967Z] [0m  21-526EZ disability compensation claim form[0m
[2020-09-21T20:32:15.967Z] [0m    full-781-781a-8940-test.json[0m
[2020-09-21T20:33:23.554Z]     [32m  ✓[0m[90m fills the form[0m[31m (67092ms)[0m
[2020-09-21T20:33:23.555Z] [0m    maximal-test[0m
[2020-09-21T20:33:35.702Z] 1 accessibility violation was detected
[2020-09-21T20:33:35.702Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-21T20:33:35.702Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-21T20:33:35.702Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-21T20:33:35.702Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   2   │
[2020-09-21T20:33:35.702Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
[2020-09-21T20:33:37.581Z]     [31m  1) fills the form[0m
[2020-09-21T20:33:37.581Z] [0m    maximal-bdd-test[0m
[2020-09-21T20:33:37.581Z]     [36m  - fills the form[0m
[2020-09-21T20:33:37.581Z] [0m    minimal-test[0m
[2020-09-21T20:33:52.398Z]     [32m  ✓[0m[90m fills the form[0m[31m (13188ms)[0m
[2020-09-21T20:33:52.398Z] [0m    minimal-bdd-test[0m
[2020-09-21T20:33:52.398Z]     [36m  - fills the form[0m
[2020-09-21T20:33:52.398Z] [0m    newOnly-test[0m
[2020-09-21T20:34:02.322Z] 1 accessibility violation was detected
[2020-09-21T20:34:02.322Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-21T20:34:02.322Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-21T20:34:02.322Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-21T20:34:02.322Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   3   │
[2020-09-21T20:34:02.322Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
[2020-09-21T20:34:04.826Z]     [31m  2) fills the form[0m
[2020-09-21T20:34:04.826Z] [0m    secondary-new-test.json[0m
[2020-09-21T20:34:16.973Z] 1 accessibility violation was detected
[2020-09-21T20:34:16.973Z] ┌─────────┬──────────────────┬───────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────┬───────┐
[2020-09-21T20:34:16.973Z] │ (index) │        id        │  impact   │                                                description                                                │ nodes │
[2020-09-21T20:34:16.973Z] ├─────────┼──────────────────┼───────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────┼───────┤
[2020-09-21T20:34:16.973Z] │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds' │   1   │
[2020-09-21T20:34:16.973Z] └─────────┴──────────────────┴───────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────┴───────┘
```

Reply:
Disabling those tests doesn't seem incredibly reliable though, so I'm going to look into modifying the cypress helpers that the platform provides so that they can opt out of the axe checks.

Reply:
Ahh, yes. I had to disable those contrast errors when I updated axe for Nightwatch

Reply:
It looks like two rules were throwing failures:
* `color-contrast` which is probably a false positive. Those are tough to get right in headless environments and I'd be okay suppressing this rule.
* `duplicate-id` is more problematic. Some ARIA pairings rely on unique IDs to map themselves properly. There was an issue ticket for this, do you have any recollection @Mottie ?

Reply:
It _should_ have been fixed, but I added a comment that I was still seeing it - https://github.com/department-of-veterans-affairs/va.gov-team/issues/12674